plugins/modules/cloudformation_stack_set.py

```diff
@@ -705,10 +730,21 @@ def main():
             changed = True
             cfn.create_stack_instances(
                 StackSetName=module.params["name"],
-                Accounts=list(set(acct for acct, region in new_stack_instances)),
-                Regions=list(set(region for acct, region in new_stack_instances)),
+                Accounts=list(set(acct for acct, _ in new_stack_instances)),
```

Comment:
Using `_` for a dummy variable is not allowed by the sanity tests. Use `dummy` instead.
```suggestion
                Accounts=list(set(acct for acct, dummy in new_stack_instances)),
```